### PR TITLE
Add notice time to translation file

### PR DESCRIPTION
**Pull request by Blueberryy**
- [ x] I agree to license my contribution under LGPL-3.0 **or** my contribution is from another project with a license compatible with LGPL-3.0

To test this pull request, follow the [instructions in the wiki](https://github.com/ajayyy/SponsorBlock/wiki/Testing-a-Pull-Request).

***

### CODE REVIEW

**Comment by Blueberryy** [comment]
don't know how to make this work, if anyone knows just give me snippet there

File: src/components/NoticeComponent.tsx [hunk 1]
```diff
@@ -195,7 +195,7 @@ class NoticeComponent extends React.Component<NoticeProps, NoticeState> {
                         id={"skipNoticeTimerText" + this.idSuffix}
                         key="skipNoticeTimerText"
                         className={this.state.countdownMode !== CountdownMode.Timer ? "hidden" : ""} >
-                            {this.state.countdownTime + "s"}
+                            {chrome.i18n.getMessage("NoticeTimeAfterSkip").replace("{seconds}", this.state.countdownTime)}
```
Comment (mini-bomba) [inline src/components/NoticeComponent.tsx:198]: We're using TS, so the `replaceValue` (second argument) to `.replace()` needs to be a string, but `countdownTime` is a number. Adding `.toString()` should make TS happy. Alternatively you can wrap the whole number value in `String()` - not sure which one is better here.
```suggestion
                            {chrome.i18n.getMessage("NoticeTimeAfterSkip").replace("{seconds}", this.state.countdownTime.toString())}
```

Comment (Blueberryy) [reply src/components/NoticeComponent.tsx]: Thank you! I pushed your changes just now

**Comment by ajayyy** [comment]
nice thanks

**Pull request merged**